pkg/identity/service.go

```diff
@@ -172,6 +173,8 @@ func (svc *Service) Certify(ctx context.Context, req *pb.CertifyRequest) (*pb.Ce
 
 	if err := svc.ensureIssuerStillValid(); err != nil {
 		log.Errorf("could not process CSR because of CA cert validation failure: %s", err)
+		message := err.Error()
+		svc.recordEvent(v1.EventTypeWarning, eventTypeFailed, message)
```

Comment:
I think it makes sense to include some infomation, indicating where the CSR is coming from. If the validation fails, then we would like to know which service actually issued the CSR request. Maybe the `Identity` field of the CSR is a good source for that information

Reply:
I think the preceding line 169 does this:
```go
	reqIdentity, tok, csr, err := checkRequest(req)
	if err != nil {
		return nil, status.Error(codes.InvalidArgument, err.Error())
	}
```
We could use `reqIdentity` in the message